### ISSUE TITLE
accessibility problem with screen reader NVDA

### ISSUE THREAD

**Issue by lstar4ever** [Lusito/forget-me-not#71]
Hi,
This is a screen reader user who rely on NVDA to navigate the web.
I found that the add on panel is not accessible via NVDA keyboard control.
e.g. 
I am able to activate the addon via toolbar icon,
however if in the setting page, screen reader cannot identify whether those item are checked or not, and pressing space has no effect.
Please help to fix.

**Comment by Lusito** [comment]
Thanks for the report. I actually didn't think about that.
I will try to get a fix for this as soon as possible.. although it might take a couple of tries to make it fully accessible, as I've never done this before.

I'm guessing you can't even navigate the tabs in the dialog panel?
I put this on the list for the UI redesign (#67), but will start working on this before everything else!

**Comment by Lusito** [comment]
@lstar4ever I've just released a new version which should at least give basic support. I've tried it with nvda.. it's a little confusing to me, as I've never used nvda, but I was able to access all fields.

If you find anything to be too confusing, please report it here and I'll try my best to improve the situation.

**Comment by Lusito** [comment]
Closing this, as I've added basic support, but have not received feedback.
If you or someone else is interested in providing more feedback, please reopen or create a new ticket.
